### ISSUE TITLE
Operators and elementwise multiplication

### ISSUE THREAD

**Issue by ckolbPTB** [PTB-MR/mrpro#247]
I am trying to create an acquisition model for Golden Radial Phase Encoding (GRPE) and then using the pseudo-inverse (i.e. adjoint + dcf) to reconstruct and image x0. I tried three different cases of how to combine the operators with the dcf but only one of them works. The other two fail because the operators are always applied first and the dcf is then applied last, which means trying to apply a dcf onto an image.

```
# WORKS
AW = kdcf.data[:,None,...] * cart_sampling_op @ fourier_op @ csm_op
x0 = AW.H(kdata_resp_resolved.data)[0]

# FAILS
A = cart_sampling_op @ fourier_op @ csm_op
AW = kdcf.data[:,None,...] * A
x0 = AW.H(kdata_resp_resolved.data)[0]

# FAILS
A = cart_sampling_op @ fourier_op @ csm_op
AHW = A.H * kdcf.data[:,None,...]
x0 = AHW(kdata_resp_resolved.data)[0]
```
I am not sure if this is an expected outcome or if this is a bug. Any input welcome!

**Comment by fzimmermann89** [comment]
yes, i know, forgot to push the fix for this issue....

**Comment by fzimmermann89** [comment]
I changed the title as this concerns element wise multiplication.

Matrix Multiplication of an Operator is not directly supported, you need to wrap the Matrix as a Linear Operator, for example the einsum op and specify how to perform the multiplication

**Comment by fzimmermann89** [comment]
Mypy will still cause issues due to pytorcs typehints on the multiplication.

In #183 I also added an overload to DCF data to be able to write
dcf@fourierop
no sure if this is nice, as it will only allow for left multiplication...

**Comment by fzimmermann89** [comment]
solved by #262
